### DISCUSSION TITLE
How to add and remove option

### DISCUSSION THREAD

**Discussion by xmaxmex** [mikefarah/yq#1848] [Q&A]
How to add a 'record: true' on these config.

And how delete 'record: true' ??

Thanks in advance

path:

  testeopencv:
          runOnDemand: camera.py admin:admin@123 192.168.5.137 testeopencv
          runOnDemandRestart: yes
          runOnDemandStartTimeout: 10s
          runOnDemandCloseAfter: 10s

  B8E81A0F-498E:
          runOnDemand: camera.py admin:admin@123 192.168.5.137 B8E81A0F-498E
          runOnDemandRestart: yes
          runOnDemandStartTimeout: 10s
          runOnDemandCloseAfter: 10s

**Comment by mikefarah** [comment]
Not sure where you want to add record: true, but you do it like this:

```
yq '.path.testeopencv.record = true' file.yaml
```

similarly to delete it:

```
yq 'del(.path.testeopencv.record)' file.yaml
```
